### ISSUE TITLE
There is no formatter for 'latte' files installed.

### ISSUE THREAD

**Issue by myalban** [kasik96/VS-Latte#28]
Hi,

If I try to format the document, VS code trigger this error : "There is no formatter for 'latte' files installed."
So, I've to choose HTML, format, then go back to latte.
Do you know a better process ?

Thank for the help !

**Comment by EncryptSL** [comment]
This extension is only for syntax colorization, there not for now included formatting.

I used Beautify from VSC marketplace. You can also try but there was problem with readable of code.

https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify

But if @kasik96 add formatting of Latte, is on him.

**Comment by geerdes** [comment]
Yes, I also need the formatter. And also autoclosing html tags. Maybe you can just Include the html formatter in a way?

**Comment by azadshaikh** [comment]
I also need the formatter. Please consider adding this feature.

**Comment by a1ternat1ve** [comment]
Yep, would be better with formatter :-)

**Comment by kasik96** [comment]
We have issue already: https://github.com/kasik96/VS-Latte/issues/10
